data/json/bionics.json

```diff
@@ -536,8 +536,11 @@
         "occupied_bodyparts": [
             [ "TORSO", 10 ]
         ],
-        "act_cost": 50,
-        "description": "You are equipped with bionic stimulators which augment your haematopoiesis system, allowing you to accelerate white blood cell production using bionic power."
+        "toggled": true,
+        "act_cost": 10,
+        "react_cost": 1,
+        "time": 600,
```

Comment:
Tried to keep it relatively similar to what it is in master and avoid exploits.

The problem is, health checks happen only once per 6 hours (3600 turns). It shouldn't be possible to activate the bionic just for the check to save power. So activation cost should be no less than the cost of keeping it active for 6 hours.

Then finally, bypassing the health system doesn't really mean that much. If the CBM limit was turned on right now, leukocyte wouldn't be worth it, except for those who get irradiated all the time.